cockatrice/src/user_context_menu.cpp

```diff
@@ -89,6 +94,27 @@ void UserContextMenu::banUser_processUserInfoResponse(const Response &r)
     dlg->show();
 }
 
+void UserContextMenu::adjustMod_processUserResponse(const Response &resp, const CommandContainer &commandContainer)
```

Comment:
So what happens.  The command is sent, the server processes it and the server tries to send you the response back but if you get disconnected the response container isnt sent at all and well it doesnt matter in the end.